underscore.js

```diff
@@ -84,18 +84,23 @@
     };
   };
 
+  var cb, builtinIteratee;
```

Comment:
Since both functions reference each other, I had do these definitions first avoid triggering ESLint's [no-use-before-define](http://eslint.org/docs/rules/no-use-before-define).